UPGRADING

```diff
@@ -280,6 +280,9 @@ PHP 7.1 UPGRADE NOTES
   . Timezone initialization failure from serialized data will now throw an
     instance of Error from __wakeup() or __set_state() instead of resulting in
     a fatal error.
+  . DateTime instances now support microseconds. Please note this can have impact
+    when comparing; eg. this would previously return TRUE, but now returns FALSE:
+     (new DateTime('first day of next month') == new DateTime('first day of next month'));
```

Comment:
It is definitely a gotcha for faulty code that happened to work most of the times.  The good thing is the bug will surface more often and get fixed!

Note in UPGRADING for 7.1 has been added with 69a3659d416b3ee74ed4a661b3eec414f18acc33